ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/PaymentDetailsForm.tsx

```diff
@@ -0,0 +1,83 @@
+import {useNavigation} from '@react-navigation/native';
+import React from 'react';
+import {useForm, SubmitHandler} from 'react-hook-form';
+import {Text, View, Pressable} from 'react-native';
+
+import CustomAmountInput from './components/customAmountInput/CustomAmountInput';
+import CustomButton from './components/customButton/CustomButton';
+import CustomInput from './components/customInput/CustomInput';
+import CustomMessageInput from './components/customMessageInput/CustomMessageInput';
+import {styles} from './payment-details-form.styles';
+import {PaymentDetailsFormData} from './payment-details-form.types';
+import {NavigationType} from '../../../navigation/navigation.types';
+
+const PaymentDetailsForm: React.FC = () => {
+  const navigation = useNavigation<NavigationType>();
+
+  const {
+    control,
+    handleSubmit,
+    formState: {errors},
+  } = useForm<PaymentDetailsFormData>({
+    defaultValues: {
+      ToPayee: 'John McCabe',
+      statementMessage: '',
+      payeeMessage: '',
+    },
+  });
+
+  const onSubmit: SubmitHandler<PaymentDetailsFormData> = data =>
+    console.log(data);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.inputBox1}>
+        <CustomInput
+          name="ToPayee"
+          title="To"
+          placeholder="Jack"
+          control={control}
+        />
+      </View>
+      <View style={styles.inputBox2}>
+        <CustomAmountInput
+          name="amountToSend"
+          title="Amount to send"
+          placeholder="0.00"
+          control={control}
+        />
+      </View>
+
+      <View>
+        <Text style={styles.paymentMessages}>Payment messages</Text>
+      </View>
+
+      <CustomMessageInput
+        name="statementMessage"
+        title="Your statement message"
+        placeholder="none"
+        control={control}
+        maxLength={12}
+      />
+      <CustomMessageInput
+        name="payeeMessage"
+        title="Payee message"
+        placeholder="none"
+        control={control}
+        maxLength={18}
+      />
+      <View style={styles.buttonContainer}>
+        <CustomButton
+          title="Continue"
+          onPressFunction={handleSubmit(onSubmit)}
+        />
+      </View>
+      <Pressable
```

Comment:
Could you please share, is there a reason we using `CustomButton` for the first `button`, but `Pressable` for the second one?
I would suggest to unify approach, having only one `Button` component, but with different `type/preset`.
This way we are not going to be confused with different representation in code for same functionality.